docs/source/release_notes/release-notes.rst

```diff
@@ -42,14 +42,36 @@ develop
     *    - Type
          - Change
 
+    *    -
+         - 
+
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+=======
+v0.39.0
+=======
+
+19 Apr 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
     *    - |improved|
-         - Refactored ``AvailableTimestamps`` reducing overzealous synchronization. Giving out timestamps is no longer blocking on refreshing the timestamp bound if there are enough timestamps to give out with the current bound.
+         - Refactored ``AvailableTimestamps`` reducing overzealous synchronization.
+           Giving out timestamps is no longer blocking on refreshing the timestamp bound if there are enough timestamps to give out with the current bound.
+           This improves latency of timestamp requests under heavy load; we have seen an approximately 30 percent improvement on internal benchmarks.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1783>`__)
 
     *    - |improved|
          - The lock server now logs to ``SlowLockLogger`` logger if a request takes more than a given time (10 seconds by default) to be processed.
```

Comment:
I've refined the precision of the first statement. Note that we only log _if a lock request receives a response after more than a given amount of time_ - if a server acquires a lock for an hour but it's the only client for that lock, we don't log anything (I think), looking at the PR.

I've removed the line on DEBUG; it's not really related to this change, though it is part of the behaviour if the parameter is 0 or negative.